.github/workflows/ci.yml

```
@@ -1,38 +1,80 @@
+
+  windows-smoke:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: VSPreview console smoke (cp1252)
+        shell: pwsh
+        run: |
+          chcp 1252 | Out-Null
+          $scriptPath = Join-Path $env:RUNNER_TEMP "vspreview_console_smoke.py"
+          @'
+import os
```

actionlint (1.7.8)

could not parse as YAML: could not find expected ':'

(syntax-check)